packages/frontend/package.json

```
@@ -1,25 +1,35 @@
 {
   "name": "eslint-config-trybe-frontend",
-  "version": "1.2.3",
+  "version": "1.3.0",
   "description": "ESLint shared config used by Trybe on frontend projects",
   "main": "config.json",
   "keywords": [
     "eslint",
     "trybe",
     "frontend"
   ],
-  "author": "Roz <roz@rjmunhoz.me>",
+  "author": "Equipe Front",
   "license": "GPL-3.0",
-  "dependencies": {
-    "babel-eslint": "^10.1.0",
-    "eslint": "6.8.0",
-    "eslint-config-airbnb": "^18.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-react": "^7.21.5",
+  "devDependencies": {
+    "eslint": "^8.20.0",
```

```suggestion
    "eslint": "8.20.0",
```

Vamos lockar a versão de todas as dependências @robertotcestari ? 
Pra no tempo não dar uns bugs bizarros, pq esse `^` faz o npm instalar qualquer versão dentro da major, só que a gente sabe que os pacotes vem sempre respeitam o versionamento arrisca

---

@robertotcestari o ponto trazido pelo @nato-re é interessante. 

Atualmente não temos versões travada em nenhuma dependência, mas acho que pode ser interessante ter essa trava. Assim a gente meio que garante que o essas libs vão ser sempre as mesmas em todos os projetos. 

Apesar de a gente nunca ter tido problemas com isso (que eu lembre), a chance de dar algum problema inesperado é menor.

---

Aceito @renatofreire @nato-re !